website/docs/opengl/2-helloQuad.md

```diff
@@ -93,9 +82,9 @@ _gl.Clear(ClearBufferMask.ColorBufferBit);
 
 Run your application again, and you should see a lovely sky blue window!
 
-![Sky blue window](../../../images/opengl/chapter1/cornflower-window.png)
+![Sky blue window](assets/2-helloQuad/cornflower-window.png)
 
-Congrats! You've done your first thing in OpenGL! Didn't work? Check the [source code](../sources/1.2.2-clear-window.html) for this section here.
+Congrats! You've done your first thing in OpenGL!
```

Comment:
We should probably have the source code at this stage linked on the website somehow